core/internal/cltest/cltest.go

```diff
@@ -1548,6 +1549,10 @@ func NewTxmORM(t *testing.T, db *sqlx.DB, cfg pg.LogConfig) txmgr.ORM {
 	return txmgr.NewORM(db, logger.TestLogger(t), cfg)
 }
 
+func NewFwdMgrORM(t *testing.T, db *sqlx.DB, cfg pg.LogConfig) forwarders.ORM {
```

Comment:
we've been trying to move stuff out of cltest, problem being it imports everything so creates import cycles preventing you from doing whitebox testing. Should be fine with just ```forwarders.NewORM(db, logger.TestLogger(t), cfg)```